spec/peeps_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'peeps'
+
+describe Peeps do
+
+  # describe '.initialize' do
+  #   it 'will have an empty array upon initlisation' do
+  #     expect(Peeps.peep_list).to be_an(Array)
+  #     expect(Peeps.peep_list).to be_empty
+  #   end
+  # end
+
+  describe '.add_peep' do
```

Comment:
I think .add_peep should be .post_peep? Guessing you changed the name of the method at some point!

Reply:
Ah yeah, great catch.